docs/specification/draft/client/elicitation.mdx

```diff
@@ -44,23 +51,120 @@ Applications **SHOULD**:
 ## Capabilities
 
 Clients that support elicitation **MUST** declare the `elicitation` capability during
-[initialization](/specification/draft/basic/lifecycle#initialization):
+[initialization](../basic/lifecycle#initialization):
 
 ```json
 {
   "capabilities": {
-    "elicitation": {}
+    "elicitation": {
+      "form": {},
+      "url": {}
+    }
   }
 }
 ```
 
+Clients declaring the `elicitation` capability **MUST** support at least one mode (`form` or `url`).
+
+Servers **MUST NOT** send elicitation requests with modes that are not supported by the client.
+
 ## Protocol Messages
 
-### Creating Elicitation Requests
+### Elicitation Requests
+
+To request information from a user, servers send an `elicitation/create` request.
+
+All elicitation requests **MUST** include the following parameters:
+
+| Name      | Type   | Options       | Description                                                        |
+| --------- | ------ | ------------- | ------------------------------------------------------------------ |
+| `mode`    | string | `form`, `url` | The mode of the elicitation.                                       |
+| `message` | string |               | A human-readable message explaining why the interaction is needed. |
+
+The `mode` parameter specifies the type of elicitation:
+
+- `"form"`: In-band structured data collection with optional schema validation. Data is exposed to the client.
+- `"url"`: Out-of-band interaction via URL navigation. Data is **not** exposed to the client.
```

Comment:
Technically, you would have some data exposed to the client - the URL. Worth calling that out.